lib/providers/add_excercise_provider.dart

```diff
@@ -2,9 +2,31 @@ import 'package:flutter/foundation.dart';
 
 import 'dart:io';
 
+import 'package:wger/models/exercises/muscle.dart';
+
 class AddExcerciseProvider with ChangeNotifier {
   List<File> get excerciseImages => [..._excerciseImages];
   final List<File> _excerciseImages = [];
+  String? _name;
+  String? _alternativeName;
```

Comment:
the alternative names should be a string since some exercises can have several names

Reply:
It's already string!

Reply:
Sorry, I meant a list of strings

Reply:
Every string seperated by space is new name. Correct?

Reply:
Either space or newline, yes. I think newline is more obvious since some names can have whitespaces in them

Reply:
I need to think,  how we can implement these.

Reply:
These part is still pending, but branch is merged?